assets/js/tour.js

```diff
@@ -99,13 +99,17 @@ jQuery( document ).ready(
 		window.loadTour = function() {
 			var color1 = '';
 			var color2 = '';
-			var sheet = {};
+			var styleElement = {};
 			var n;
+			var style;
 			for ( n in window.tour ) {
 				color1 = window.tour[ n ][ 0 ].color + '00';
 				color2 = window.tour[ n ][ 0 ].color + 'a0';
-				sheet = document.styleSheets[ 0 ];
-				sheet.insertRule( '@keyframes animation-' + n + ' {' +
+				styleElement = document.createElement( 'style' );
```

Comment:
Do we really have to create multiple stylesheets? Why not create one before starting the loop?